.github/workflows/ci.yml

```diff
@@ -12,13 +12,17 @@ jobs:
         with:
           toolchain: 1.56.1
           components: rustfmt, clippy
+      # Pinned to the commit hash of v2.2.1
+      - uses: Swatinem/rust-cache@6fd3edff6979b79f87531400ad694fb7f2c84b1f
```

Comment:
Could you put the context of why we use the commit hash of v2.2.1?

Reply:
And may I have benefits of [Swatinem/rust-cache](https://github.com/Swatinem/rust-cache) over [actions/cache](https://github.com/actions/cache). As far as I see, the actions/cache also supports rust. 
https://github.com/actions/cache#implementation-examples

Reply:
@StoneDot Thanks for your input! So it's sticking the latest version of Swatinem/rust-cache, isn't it?
https://github.com/Swatinem/rust-cache/releases

I believe this kind of historical information would be helpful for our future. I'd appreciate it if you could put it into the commit message.

Reply:
> And may I have benefits of Swatinem/rust-cache over actions/cache (github official one)?

I have two reasons to chose Swatinem/rust-cache instead of actions/cache.

1. We can use and maintain the cache easily rather than manual configuration of actions/cache also it has more function.
2. Already used by aws-sdk-rust

Reply:
Fair enough. I hope this discussion would be a future reference. Thanks!